client/core/simnet_trade.go

```diff
@@ -2059,17 +2059,33 @@ func (client *simulationClient) findOrder(orderID string) (*trackedTrade, error)
 func (client *simulationClient) disableWallets() {
```

Comment:
I wanted to get rid of this, but it looks needed for some tests like `testNoMakerSwap`